content/docs/guide/map_submission/map_zoning.md

```diff
@@ -9,60 +9,420 @@ weight: 4
+{{<hint warning>}}
+
+This guide applies only to maps that require courses to end **before** the start of next course.  
+If the next course starts at the same spot where the previous course ends, **don't change the main track property** and **don't add checkpoints** to the end of **courses**.   
+If you have any questions please ask in **#map-porting** channel on our [Discord](https://discord.gg/momentummod).
+
+{{</hint>}}
+
+1. Uncheck **End segment at next segment start** in **main track** properties
+    - This will make the time submit when the player enters **last checkpoint** on a **course** except for the **last course**
+    - **Last course** time will **always** submit when player enters the [End Zone](/guide/map_submission/map_zoning/#creating-the-end-zone)
+![RJ/SJ Courses Properties](/images/map_zoning/zoning_rocket_sticky_jump_courses_properties.png)
+2. For every **course except last**:
+    1. Create a **new segment** at the first jump
+    2. Add a **checkpoint** to the start of every jump
+    3. Add a **checkpoint** to the end platform ( before being teleported to the hub )
+3. For the **last course**:
+    1. Create a **new segment** at the first jump
+    2. Add a **checkpoint** to the start of every jump
+        - **Do not** add a checkpoint to the end of this course
+
+**Example: rj_deserted zones** ( notice that the end of **course 1** is zoned with a **checkpoint** )
+![RJ/SJ Courses](/images/map_zoning/zoning_rocket_sticky_jump_courses.png)
+
+### Other Gamemodes
+Vast majority of maps for gamemodes other than **Surf** and **Rocket / Sticky Jump** will use a **single stage** with **checkpoints**.  
+That means they should be zoned identically to [Surf: Linear](/guide/map_submission/map_zoning/#surf-linear) maps.  
+
+{{<hint info >}}
+
+The only difference is that for modes other than Surf the **Limit ground speed** checkbox will be automatically **unchecked**.  
+That is the correct behavior so you don't have to worry about it.
+
+{{</hint>}} 
+
+{{<hint warning>}}
+
+When zoning **Defrag** maps, also make sure to make checkpoints [not required](#required-checkpoints)
+
+{{</hint>}}
+
+{{<hint warning>}}
+
+Of course there are exceptions to this rule.  
+There is nothing stopping maps for other gamemodes to be staged.  
+The reason they are not is simply because of the way different communities create maps.  
+Some exceptions are:
+- **bhop_sqee** ( Multiple long courses. Every course is a seperate stage because they are fun to compete on individually )
+- **df_parkourushi** ( Long parkour courses. In defrag due to technical limitations every course was made to be a seperate map. In Momentum Mod they can be stages )   
+
+In case you encounter these exceptions, apply your best judgement to which method of zoning from above to use.  
+When in doubt, don't hesitate to ask for help in **#map-porting** channel on our [Discord](https://discord.gg/momentummod).
```

Comment:
I'm not familiar with Defrag maps, but I think for bhop at least the exceptions that work are so rare (if we even want to allow them) that this is not worth mentioning and it will only lead to people submitting things with stages that definitely shouldn't have them.